prow/cmd/deck/static/prow/prow.ts

```diff
@@ -653,16 +654,28 @@ function redraw(fz: FuzzySearch): void {
 function createRerunCell(modal: HTMLElement, rerunElement: HTMLElement, prowjob: string): HTMLTableDataCellElement {
     const url = `https://${window.location.hostname}/rerun?prowjob=${prowjob}`;
     const c = document.createElement("td");
-    const i = icon.create("refresh", "Show instructions for rerunning this job");
-    i.onclick = () => {
+    let i;
+    if (rerunCreatesJob) {
+      i = icon.create("refresh", "Rerun this job");
+      i.onclick = () => {
```

Comment:
Just a heads up - some people have said they'd still like to see the "kubectl create ..." popup, even if directly triggering the rerun is enabled. In my [PR](https://github.com/kubernetes/test-infra/pull/13008), I'm adding a "run" button within that popup that will actually rerun the job, so it's still displayed:
![image](https://user-images.githubusercontent.com/28876252/59721107-67f9da80-91d5-11e9-9647-f889c8063e49.png)

Reply:
@mirandachrist If you just hadn't explained that, I wouldn't have derived from the view on the screenshot that the run button runs the job tbh (But that discussion is probably better put on your PR)